client/src/components/Calendar/StaticCalendar.js

```diff
@@ -6,281 +6,122 @@ function Timeslot(props) {
   return <div class="item-cell" style={{ backgroundColor: props.color }}></div>;
 }
 
+class Shift {
+  constructor(color, id, start, end, day) {
+    this.color = color;
+    this.id = id;
+    this.start = start;
+    this.end = end;
+    this.day = day;
+  }
+}
+
+function initialShifts() {
+  let a = [];
+  for (var i = 0; i < 168; i += 1) {
+    a.push(new Shift("#f8f8f8", null, null, null, null));
+  }
+  let count = 0;
+  for (var i = 0; i <= 23; i += 1) {
+    for (var j = 0; j <= 6; j += 1) {
+      a[count].start = i;
+      a[count].end = i + 1;
+      a[count].day = j;
+      count += 1;
+    }
+  }
+  return a;
+}
+
+var currentDate = new Date();
+var weekString =
+  format(currentDate, "MMMM") +
+  " " +
+  format(currentDate, "YYYY") +
+  ": " +
+  format(startOfWeek(currentDate), "MM/DD") +
+  " - " +
+  format(endOfWeek(currentDate), "MM/DD");
+
 export default class StaticCalendar extends React.Component {
   constructor(props) {
     super(props);
-    var a = new Array(24);
-    for (var i = 0; i <= 23; i += 1) {
-      a[i] = new Array(7);
-    }
-    this.state = { grid: a };
-    this.currentDate = new Date();
-    this.weekString =
-      format(this.currentDate, "MMMM") +
-      " " +
-      format(this.currentDate, "YYYY") +
-      ": " +
-      format(startOfWeek(this.currentDate), "MM/DD") +
-      " - " +
-      format(endOfWeek(this.currentDate), "MM/DD");
+    this.state = { shifts: initialShifts() };
   }
 
   componentDidMount() {
-    console.log("mount");
-    fetch("/staticcalendar")
+    fetch("/staticcalendar", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({ items: this.state.shifts })
```

Comment:
Why are we sending the shifts to the backend? Aren't we just getting the shifts from the backend at this endpoint?